lib/utils.pm

```diff
@@ -266,7 +266,8 @@ sub assert_gui_app {
 # console font, we need to call systemd-vconsole-setup to workaround
 # that
 sub check_console_font {
-    return if check_var('BACKEND', 'spvm');
+    # cover ipmi case now
```

Comment:
can you please rephrase this to something like
`Does not make sense on ssh-based consoles`

Reply:
sure, updated, thanks!